packages/nns/README.md

```diff
@@ -33,28 +26,23 @@ npm i @dfinity/agent @dfinity/candid @dfinity/principal @dfinity/utils @dfinity/
 
 ## Usage
 
-Most features are provided through the use of classes. e.g. querying the `ledger` for an account balance in a NodeJS context can be developed as following:
+Most features are provided through the use of classes. For example, querying the list of neurons controlled by the caller with the `governance` canister:
 
 ```ts
-import { AccountIdentifier, LedgerCanister } from "@dfinity/nns";
+import { GovernanceCanister } from "@dfinity/ledger-icrc";
```

Comment:
I thes this is a typo `@dfinity/ledger-icrc` should be `@dfinity/nns`, right?

Reply:
Absolutely!!! Thanks.